docs/source/overview/options.md

```diff
@@ -144,6 +144,9 @@ find more details in [Olive Models](https://microsoft.github.io/Olive/api/models
             ```
             For cases where you do not want to use the huggingface model but want to use the huggingface dataset, you can provide `dataset` config only like above.
 
+        - `model_loading_args: [dict]`: Arguments to pass to the `from_pretrained` method of the model class. Refer to [this documentation](https://huggingface.co/docs/transformers/main_classes/model#transformers.PreTrainedModel.from_pretrained).
```

Comment:
Folks are familiar with HF's from_pretrained() API, so rename it to be 'hf_from_pretrained_args' for clarity.

Reply:
Sure, will send a separate PR for this change.

Reply:
PR link: https://github.com/microsoft/Olive/pull/785